requirements.txt

```diff
@@ -15,7 +15,7 @@
 # numpy==1.13.3
 # scipy==1.0
 #
-eiffellib==1.1.0
+eiffellib==1.1.1
```

Comment:
This probably belongs in a separate commit. Besides, isn't this file autogenerated or something? At the very least the `install_requires` directive in setup.cfg should be updated in tandem?

Reply:
Indeed, just changing requirements.txt doesn't affect what ends up in the dependencies of the package that you build from setup.py. I've verified that 1.1.1 solves my consumption problem so I'll send a PR.